src/ODM/MongoDB/Types/CurrencyType.php

```diff
@@ -2,10 +2,10 @@
 
 namespace ZFBrasil\DoctrineMoneyModule\ODM\MongoDB\Types;
 
-use Doctrine\ODM\MongoDB\Types\StringType;
+use Doctrine\ODM\MongoDB\Types\Type;
 use Money\Currency;
 
-class CurrencyType extends StringType
+class CurrencyType extends Type
```

Comment:
Well, in DB side it is a string, so I think it's ok to inherit VO -> string conversions from `StringType`. Just a suggestion, though.